Library/Homebrew/brew.rb

```diff
@@ -17,6 +17,16 @@
 # Also define here so we can rescue regardless of location.
 class MissingEnvironmentVariables < RuntimeError; end
 
+def create_tap(cmd, tap_name)
```

Comment:
Why make this a single use function in the same file? It adds code and indirection for no benefit. Additionally, `HOMEBREW_BREW_FILE` and others have not yet been defined at the function definition time.

Reply:
Weird, I had no issue with that when testing this. Not sure to understand what you mean by "not defined".

Reply:
If the method is called before `require "global"` below it will fail. It's moot as I don't think a function should be declared here but I think it should be declared after the `require` due to this.